handbook/ops/bizops/analytics.md

```diff
@@ -0,0 +1,62 @@
+# Analytics
+
+This page describes Sourcegraph's analytics function, our data sources, and how to use our data tools.
+
+- [Data sources](#data-sources)
+- [Data tools/tech stack](#data-tools)
+- [Using Looker](#using-looker)
+
+## Data sources
+
+We collect data from the following:
+
+- Google Analytics: Website analytics for Sourcegraph marketing and docs pages (not Sourcegraph.com)
+- HubSpot: Marketing automation
+- Salesforce: Customer Relationship Management system (CRM)
+- MixMax: Email marketing automation (Apollo is not used in production, but still retains data)
+- ZoomInfo:  Data enrichment of account and contact information
+- Sourcegraph.com Site-admin pages: customer subscriptions and license keys
+- [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information. There are [specific guidelines](https://docs.sourcegraph.com/dev/background-information/adding_ping_data) that must be followed for teams to add ping data. 
+- [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
```

Comment:
@ebrodymoore is this referring to eventLogger? 

If so, this collects events on _both_ local instances and sourcegraph.com (see the [backend function for logEvent](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/usagestats/event_handlers.go?utm_source=VSCode-1.2.3#L42-54), which collects events for both). The difference is that Sourcegraph.com events go to BigQuery, whereas on an instance it goes to the local postgres database.

```suggestion
- [Event logger: custom tool to track events](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/tracking/eventLogger.ts). On Sourcegraph.com, this sends events directly to BigQuery. On customer instances, this sends events to the `EventLogs` database, which is then used to populate pings.
```

Reply:
Thanks! This actual content hasn't been updated in a looong time so please update anything else you see